src/main/java/graphql/GraphQL.java

```diff
@@ -188,11 +191,58 @@ public ExecutionResult execute(String requestString, Object context, Map<String,
     }
 
     public ExecutionResult execute(String requestString, String operationName, Object context, Map<String, Object> arguments) {
```

Comment:
yes, let us break the `ExecutionStrategy` but keep the blocking `GraphQL.execute` method.